src/routes/WebsiteRoutes.jsx

```diff
@@ -17,6 +18,7 @@ const WebsiteRoutes = () => {
             <Route path="/login" element={<Login />} />
             <Route path="/mockman" element={<Mockman />} /> 
             <Route path="/" element={<LandingPage />} /> 
+            <Route path="*" element={<NotFound />} />
             <Route path="/" element={<ProtectedRoutes />}>
                 <Route path="/cart" element={<Cart />} />
                 <Route path="/wishlist" element={<WishList />} />
```

Comment:
Routing looks good to me.  You are planning to protect routes later if I am not wrong? Other than that everything is clean and readable.

Reply:
Thank you. No, I have already added private routes.

Reply:
Thank you for your review!